### PR TITLE
initial fix

### PR DESCRIPTION
**Pull request by trapacska**
original: https://app.bitrise.io/build/b0a4cde6c535a334#?tab=log
fixed: https://app.bitrise.io/build/9aeccb989fe685de#?tab=log

### CODE REVIEW

File: test/converters/xcresult3/action_invocation_record.go [hunk 1]
```diff
@@ -65,11 +67,20 @@ type Message struct {
 	Value string `json:"_value"`
 }
 
+func testCaseMatching(test ActionTestSummaryGroup, testCaseName string) bool {
+	class, method := test.references()
+
+	if testCaseName == class+"."+method ||
```
Comment (trapacska) [inline test/converters/xcresult3/action_invocation_record.go]: I know, I will move this into the return :(

File: test/converters/xcresult3/action_test_summary_group_test.go [hunk 2]
```diff
@@ -5,37 +5,37 @@ import (
 	"testing"
 )
 
-func TestActionTestSummaryGroup_producingTargetAndTestCaseName(t *testing.T) {
+func TestActionTestSummaryGroup_producingTestCaseName(t *testing.T) {
```
Comment (lpusok) [inline test/converters/xcresult3/action_test_summary_group_test.go]: Please rename test case to match the tested function.

File: test/converters/xcresult3/action_invocation_record_test.go [hunk 3]
```diff
@@ -66,7 +70,7 @@ func TestActionsInvocationRecord_failure(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := tt.record.failure(tt.test); got != tt.want {
+			if got := tt.record.failure(tt.test, junit.TestSuite{Name: "Xcode11TestUITests2"}); got != tt.want {
```
Comment (lpusok) [inline test/converters/xcresult3/action_invocation_record_test.go]: Is it possible to add an additional test case for multiple level of test cases?

File: test/converters/converters_test.go [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+		converter     Intf
+		testFilePaths []string
+		wantDetect    bool
+		wantXML       junit.XML
+		wantXMLError  bool
+	}{
+		{
+			name:          "xcresult3",
+			converter:     &xcresult3.Converter{},
+			testFilePaths: []string{filepath.Join(os.Getenv("BITRISE_SOURCE_DIR"), "_tmp/xcresults/xcresult3_multi_level_UI_tests.xcresult")},
```
Comment (lpusok) [inline test/converters/converters_test.go]: Can you add the descriptions of test cases (e.g. any specific setup like inherited classes) used in this xcresult, or any info needed to produce a similar xcresult file.

**Pull request merged**